kernel/kernel-impl/src/main/java/org/sakaiproject/authz/impl/BaseAuthzGroup.java

```diff
@@ -1077,6 +1097,13 @@ public void removeMembers()
 		if (m_lazy) baseAuthzGroupService.m_storage.completeGet(this);
 
 		m_userGrants.clear();
+		
+		//send message to (ignite) MicrosoftMessagingService
```

Comment:
Shall we try/catch this or is it safe?

Reply:
It's safe. Try-catch is done inside MicrosoftMessagingService.send